### DISCUSSION TITLE
Have nodes around a center point?

### DISCUSSION THREAD

**Discussion by pRob3** [Blazor-Diagrams/Blazor.Diagrams#276] [Q&A]
Is there a built in function to have all the nodes around a parent node or a good way to solve it?

Like this screenshot:
![fibaro nodes](https://user-images.githubusercontent.com/6283573/205431951-c2018524-b8fe-4429-a7aa-366332a38582.png)

**Comment by zHaytam** [comment]
Hello!

This needs some kind of layout algorithm. Unfortunately, I don't know which one is used in your example, but if you figure it out, you can probably implement it.

I think it's some kind of force directed algorithm, but i'm not sure.

**Reply by pRob3** [reply]
In the example I use vis.js.
Thanks for the quick answer.
